### PR TITLE
Fixes last force cast.

### PR DESCRIPTION
**Pull request by RamonGilabert**

### CODE REVIEW

File: Source/TopView/TopView.swift [hunk 1]
```diff
@@ -98,8 +98,10 @@ class TopView: UIView {
     let bundlePath = NSBundle(forClass: self.classForCoder).resourcePath?.stringByAppendingString("/ImagePicker.bundle")
     let bundle = NSBundle(path: bundlePath!)
     let traitCollection = UITraitCollection(displayScale: 3)
-    let image = UIImage(named: name, inBundle: bundle, compatibleWithTraitCollection: traitCollection)
-
-    return image!
+    if let image = UIImage(named: name, inBundle: bundle, compatibleWithTraitCollection: traitCollection) {
```
Comment (zenangst) [inline Source/TopView/TopView.swift]: But what about :guardsman: ?

Comment (vadymmarkov) [inline Source/TopView/TopView.swift]: `return UIImage(named: name, inBundle: bundle, compatibleWithTraitCollection: traitCollection) ?? UIImage()`

**Comment by zenangst** [comment]
@RamonGilabert :pizza:

File: Source/TopView/TopView.swift (hunk 1 above)
Comment (RamonGilabert) [inline Source/TopView/TopView.swift]: Done.

**Comment by RamonGilabert** [comment]
:open_mouth:

**Comment by zenangst** [comment]
Don't worry, its crustless

**Pull request merged**